skyportal/tests/frontend/test_shifts.py

```diff
@@ -102,20 +102,25 @@ def test_shift(
     select_users = '//*[@id="select-users--multiple-chip"]'
     driver.wait_for_xpath(select_users)
     driver.click_xpath(select_users)
-    driver.wait_for_xpath(f'//li[@id="select_users"]/*[@id="{user.id}"]')
-    driver.click_xpath(f'//li[@id="select_users"]/*[@id="{user.id}"]')
+    driver.wait_for_xpath(f'//li[@id="select_users"]/*[@id="{user.id}"]').click()
 
-    driver.wait_for_xpath(f'//li[@id="select_users"]/*[@id="{view_only_user.id}"]')
-    driver.click_xpath(f'//li[@id="select_users"]/*[@id="{view_only_user.id}"]')
+    driver.wait_for_xpath(
+        f'//li[@id="select_users"]/*[@id="{view_only_user.id}"]'
+    ).click()
 
     # check for button to add and remove users
     add_users_button = '//*[@id="add-users-button"]'
     driver.wait_for_xpath(add_users_button)
     driver.click_xpath(add_users_button)
 
     remove_users_button = '//*[@id="remove-users-button"]'
-    # As the component rerenders, the remove button will be deactivated for a bit, so we wait for the xpath to stay for a second to allow the button to be clickable
-    # if we checked for the xpath right now, it might disappear right after when we try to click it. So we add a little delay before clicking the button
+    # As the component rerenders,
```

Comment:
Why wrap on 54?

Reply:
I was just wrapping based on natural breaks in the text that are less than 80.